### DISCUSSION TITLE
Are vocabularies in the CDM 5.4 up to date with new releases?

### DISCUSSION THREAD

**Discussion by rosemarybliss** [OHDSI/Vocabulary-v5.0#983] [Q&A] [answered]
Hello community! I am a bit stuck on this. 

Are the vocabularies in the CDM- like ICD10 and SNOMED up to date with their respective releases? I want to create a reference list of ICD-10 concepts, but I noticed that CMS implemented 42 new diagnosis codes into the International Classification of Diseases, Tenth Revision, Clinical Modification (ICD-10-CM), effective April 1, 2023. 

Does OHDSI incorporate those later?

**Comment by TinyRickC137** [comment, marked as answer]
Dear @rosemarybliss 
I am sorry it took us so long to answer. The vocabularies are mostly up-to-date with their respective releases, except for the lag between the start of the work on the vocabulary and the release. You may experience a slight delay, usually around 2-3 months. 

To check the vocabulary version, you can download the vocabularies and see the versions in the 'vocabulary' table in the respective field.

Regarding your question, the current version of ICD10CM is already an FY2025. OMOPed version of SNOMED consists of three modules, the respective versions are: 
2024-02-01 SNOMED CT International Edition; 
2024-03-01 SNOMED CT US Edition; 
2024-04-10 SNOMED CT UK Edition

Please follow the [roadmap](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Release-planning), we publish planned vocabulary versions there.

And I also suggest using the [OHDSI forum](https://forums.ohdsi.org/) for communication. People are much more active there.
